frontend/cypress/e2e/map.cy.js

```diff
@@ -68,3 +70,49 @@ describe('Analysis should work for Livelihoods zones layer', () => {
     cy.get('#analysisPanelView').should('be.visible');
   });
 });
+
+describe('Share modal should show shorten URL', () => {
+  const shortenUrl = `http://localhost:3000/share/${shareUrlEncodeFixture.attributes.uid}`;
+
+  beforeEach(() => {
+    cy.interceptAllRequests();
+
+    cy.intercept('POST', '/api/share', {
+      fixture: 'share-url-encode.json',
+    }).as('shareEncodeRequest');
+
+    cy.intercept('GET', '/api/share/cb69ec73745eb70c4a5d', {
```

Comment:
Shouldn't you import the ID from `shareUrlEncodeFixture.attributes.uid` here as well?